.drone.star

```diff
@@ -1978,12 +1990,13 @@ def ocisServer(storage, accounts_hash_difficulty = 4, volumes = [], depends_on =
 
     if deploy_type == "wopi_validator":
         environment["GATEWAY_GRPC_ADDR"] = "0.0.0.0:9142"  # make gateway available to wopi server
+        environment["OCIS_ASYNC_UPLOADS"] = False  # disable async uploads because it could interfere with the wopi validator
```

Comment:
will this have an impact on live usage of the wopi service?

generally speaking: what breaks in ci breaks in real live as well ...

Reply:
I'll remove it.
I've checked locally and it doesn't seem to be needed for the tests. Probably wrong troubleshooting when I was making the pipeline.